### DISCUSSION TITLE
ethe date syntax doesn't show, e.g., tp.date.now("YYYY-MM-DD") doesn't show the time

### DISCUSSION THREAD

**Discussion by MingleiYang** [BryanJenksCommunity/FAQ#78] [Q&A] [answered]
Hi, Bryan,

thank you very much for sharing how you use obsidian to improve productivity. it is such a great pleasure to watch your video.

I am trying to use your templates. however, I found the date syntax, e.g., ==<%tp.date.now("YYYY-MM-DD")%>== doesn't work.

Could you give me some suggestions about how to figure this out?

PS, I have installed templater plugin already. it seems the command cannot be executed in obsidian.

My best,

Minglei

**Comment by tallguyjenks** [comment, marked as answer]
Hard to troubleshoot this but run through this checklist:

- [ ] Templater is installed
- [ ] You've restarted obsidian after this
- [ ] you've update the plugin to latest version
- [ ] in plugin settings tempalter is looking in the correct folder for your templates
- [ ] The insert plugin command being used is templater's
- [ ] if there is still an issue, try to recreate all of this behavior in the help vault (a place with no other plugins, or anything.
